src/components/list-of-implementers.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import getHyphenatedString from '../../utils/get-hyphenated-string'
+
+import Note from './note'
+
+const ListOfImplementers = ({ implementers = [], cls = ``, ruleId = `` }) => {
+	if (!implementers || !implementers.length) {
```

Comment:
```suggestion
	if (implementers.length === 0) {
```
Should be good with the `PropTypes` and the default value, and easier to understand (I think).